.github/workflows/slack-workflow-run.yml

```diff
@@ -47,7 +47,7 @@ jobs:
 
       - name: Post Slack Notification
         uses: slackapi/slack-github-action@6c661ce58804a1a20f6dc5fbee7f0381b469e001 # v1.25.0
-        if: ${{ steps.setup.outputs.message != 'nothing' }}
+        if: ${{ github.event.workflow_run.conclusion != 'skipped' }}
```

Comment:
Should it be the same for `cancelled`? Because we will get "nothing" for listed workflows if someone will cancel them.

Reply:
Ah thank you, I actually meant `cancelled`, not `skipped`. 👍

Reply:
You can update the description as well.